docs/src/main/paradox/server-class.md

```diff
@@ -1,32 +1,33 @@
-The Akka HTTP server class
---------------------------
+Server logic
+------------
 
-Let's dissect the server class, `QuickstartServer`. We make this class runnable by extending `App` (we will discuss the trait `JsonSupport` later):
+The main class, `QuickstartServer`, is runnable because it extends `App`, as shown in the following snippet. We will discuss the trait `JsonSupport` later.
 
 @@snip [QuickstartServer.scala]($g8src$/scala/com/lightbend/akka/http/sample/QuickstartServer.scala) { #main-class }
 
-Now that we have a class to run we should add some Akka HTTP fundamentals with which we will build our RESTful web service. The aim is to use routes to define endpoints by choosing what to do for which requests.
+## Binding endpoints
 
-## Routes
+Each Akka HTTP `Route` contains one or more `akka.http.scaladsl.server.Directives`, such as: `path`, `get`, `post`, `complete`, etc. There is also a [low-level API](http://doc.akka.io/docs/akka-http/current/scala/http/low-level-server-side-api.html) that allows to inspect requests and create responses manually. For the user registry service, the example needs to support the actions listed below. For each, we can identify a path, the HTTP method, and return value:
 
-For our service we want to define the following endpoints:
+|Functionality       | Path       | HTTP method     | Returns              |
```

Comment:
I'd suggest to swap method and path columns. Seems more natural to mention the verb before the object.